libwayshot/src/image_util.rs

```diff
@@ -25,16 +27,5 @@ pub(crate) fn rotate_image_buffer(
             image::imageops::rotate270(&flipped_buffer)
         }
         _ => image,
-    };
```

Comment:
This is my first time using ReviewStack so maybe I'm using it incorrectly, however, Why are we removing this buffer dimension check? If I'm not mistaken, this helps us completely remove one imageops::resize right?

Reply:
Is there a case where we are resizing? From my understanding, we're never resizing here because the dimensions are always the same.

If not, I'll revert it and add a comment saying when it's not the case so it's clear.

Reply:
I'll ping decode real quick, they might be better at explaining why it was needed in the first place. I'm not too confident in my understanding of this section.

@Decodetalkers

Reply:
this should not be moved. image captured by wlr-screencopy under different scale, the scale of the screen will be different, so we need to resize it

Reply:
I thought so too. The compositor can choose to render at a different scale than that of the true wl_output if I'm not mistaken.